snippets/freeipa_register.erb

```diff
@@ -24,6 +24,9 @@ name: freeipa_register
 #
 #   freeipa_opts                Additional options to pass directly to installer
 #
+
+/usr/sbin/sshd-keygen
```

Comment:
Yeah, I was mostly curious as to how compatible it was - it's good to know what the fallback is if we come across an OS without a decent way to regenerate.